components/VerifyCredentialResult.yml

```diff
@@ -15,27 +15,53 @@ components:
     VerificationResult:
       type: object
       additionalProperties: false
-      description: Object summarizing a verification
+      description: Object to report the result of a verification process on a VerifiableCredential.
       properties:
         verified:
           type: boolean
-          description: Overall verification assertion of the payload. This is set to True if no errors were detected during the verification process, otherwise False. See error handling section for more guidance around errors, warnings, validation and verification.
+          description: Overall verification assertion of the VerifiableCredential. This is set to True if no errors were detected during the verification process, otherwise False. See error handling section for more guidance around errors, warnings, validation and verification steps.
```

Comment:
```suggestion
          description: Overall verification assertion of the VerifiableCredential. This is set to `True` if no errors were detected during the verification process; otherwise, `False`. See error handling section for more guidance around errors, warnings, and validation and verification steps.
```